src/classes/AddressService.cls

```diff
@@ -74,25 +74,27 @@ public inherited sharing class AddressService {
     * @param addr Address
     * @return boolean. true if any of the Address fields on the Contact are different from this Address record
     */
-    public Boolean isSObjectAddressDifferent(SObject sObj, Address__c addr) {
+    //Todo: includes lat and long, wBilling, compares HHAcct not HH?
+    public Boolean isSObjectAddressDifferent(SObject sObj, IAddress other) {
+        Address__c addr = (Address__c) other.getRecord();
         if (sObj == null || addr == null) {
```

Comment:
Yeah, this is another method that should be further refactored/relocated.  A follow up WI here should refactor so that Addresses are compared using the entities' query methods, i.e., `contact.mailingAddress.isEqualTo(household.address())`
This line should be refactored along with that work.